src/object.d

```diff
@@ -3241,6 +3241,7 @@ template RTInfo(T)
     enum RTInfo = null;
 }
 
+// Compare floating point numbers for ordering (one instantiation per width).
```

Comment:
I.e. `F` will still be non-const but that's not a problem because it will always be non-const :)